### PR TITLE
feat: add `PullRequestSocialCard` component to design system

### LINKED ISSUE #716: feature: PR social cards for displaying recent work.
Screen Shot 2022-12-22 at 7 27 39 AM

This should start as a design component. 

https://www.figma.com/file/OpVX6WT7dmWqnwRuEvADMF/OpenSauced-%E2%80%94-Insights-(insights.opensauced.pizza)?node-id=5204%3A6170&t=T8av4JLONEnmJyCa-0

### PR DESCRIPTION
**Pull request by OgDev-01**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR:

- adds the `PullRequestSocialCard` component to the design system

## Related Tickets & Documents
Fixes #716 

### Current data present on card
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [x] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by brandonroberts** [comment]
@OgDev-01 there's a build failure and I'm unable to see the design system preview

File: components/molecules/PullRequestSocialCard/pull-request-social-card.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  languageList,
+  prTicketId,
+  commentsCount,
+  linesAdded,
+  linesRemoved
+}: PullRequestSocialCardProps) => {
+  return (
+    <div className="w-[26.375rem] cursor-pointer border rounded-xl px-5 py-4">
+      <div className="flex gap-2 items-center ">
+        <Avatar alt="" avatarURL={avatar} initials="" size="sm" className="" />
```
Comment (brandonroberts) [inline components/molecules/PullRequestSocialCard/pull-request-social-card.tsx:36]: Remove empty className

```suggestion
        <Avatar alt="" avatarURL={avatar} initials="" size="sm"  />
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: components/molecules/PullRequestSocialCard/pull-request-social-card.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  prTitle,
+  avatar,
+  languageList,
+  prTicketId,
+  commentsCount,
+  linesAdded,
+  linesRemoved
+}: PullRequestSocialCardProps) => {
+  return (
+    <div className="w-[26.375rem] cursor-pointer border rounded-xl px-5 py-4">
```
[the remark this answers is not in the document]
Comment (OgDev-01) [reply components/molecules/PullRequestSocialCard/pull-request-social-card.tsx:34]: That's the pixels equivalent in rem from the design, and tailwind has no size value for that.

File: components/molecules/PullRequestSocialCard/pull-request-social-card.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      </div>
+
+      {/* Language list section */}
+      <div className="mt-6">
+        <CardHorizontalBarChart withDescription={false} languageList={languageList} />
+      </div>
+
+      <div className="flex items-center gap-5 text-light-slate-10 mt-3">
+        <div className="flex justify-between gap-2">
+          {linesAdded && (
```
Comment (brandonroberts) [inline components/molecules/PullRequestSocialCard/pull-request-social-card.tsx:57]: This should still display if the `linesAdded` is 0

File: components/molecules/PullRequestSocialCard/pull-request-social-card.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            <span title="lines added" className="text-green-600">
+              +{linesAdded}
+            </span>
+          )}
+          {linesRemoved && (
```
Comment (brandonroberts) [inline components/molecules/PullRequestSocialCard/pull-request-social-card.tsx:62]: This should still display if the `linesRemoved` is 0.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

File: components/molecules/PullRequestSocialCard/pull-request-social-card.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        <Text className="!text-sm !text-light-slate-11">
+          {orgName}/<span className="text-light-slate-11">{repoName}</span>
+        </Text>
+      </div>
+      <div className="flex gap-2 items-start mt-2">
+        <IconContext.Provider value={{ color: "green", style: { width: 18, height: 18, marginTop: 4 } }}>
+          <VscGitPullRequest title="Open Pull Request" />
+        </IconContext.Provider>
+        <Text className="!text-lg ">
+          <span className="text-light-slate-10 ">{prTicketId}</span> {prTitle}
```
Comment (brandonroberts) [inline components/molecules/PullRequestSocialCard/pull-request-social-card.tsx:44]: Might need to truncate this, but can be for a follow-up

**Pull request merged**